es2016plus/index.html

```diff
@@ -5937,9 +5765,7 @@
 <td class="no not-applicable" data-browser="closure" title="This feature is optional on non-browser platforms, and doesn&apos;t contribute to the platform&apos;s support percentage.">No</td>
 <td class="no not-applicable" data-browser="typescript" title="This feature is optional on non-browser platforms, and doesn&apos;t contribute to the platform&apos;s support percentage.">No</td>
 <td class="no not-applicable" data-browser="es7shim" title="This feature is optional on non-browser platforms, and doesn&apos;t contribute to the platform&apos;s support percentage.">No</td>
-<td class="no obsolete" data-browser="ie9">No</td>
-<td class="yes obsolete" data-browser="ie10">Yes</td>
-<td class="yes" data-browser="ie11">Yes</td>
+<td class="no" data-browser="ie11">No</td>
```

Comment:
I think that in general it's important to be able to see when support for a thing started landing.

It would also be a fair policy to show the oldest browser that had nonzero support (instead of the newest with zero), as long as we're consistent.